main.tf

```diff
@@ -175,7 +175,7 @@ resource "helm_release" "hpcc" {
   disable_openapi_validation = try(local.hpcc.disable_openapi_validation, null)
   wait                       = try(local.hpcc.wait, null)
   dependency_update          = try(local.hpcc.dependency_update, null)
-  timeout                    = try(local.hpcc.timeout, 600)
+  timeout                    = try(local.hpcc.timeout, 1200)
```

Comment:
This appears to be a change unrelated to the PR.  Please revert.

Reply:
> This appears to be a change unrelated to the PR. Please revert.

Tim: Now, reverted back to "600".